x-pack/plugins/lens/public/datasources/form_based/to_expression.ts

```diff
@@ -367,6 +371,32 @@ function getExpressionForLayer(
           },
         };
 
+        if (col.operationType === 'formula') {
```

Comment:
if confuguration that was optimized is detected we don't want to do time scaling on the client (es already did that) and we just need to format the values

Reply:
```suggestion
        if(isColumnOfType<formula...>(col)){
```